release/benchmarks/distributed/dashboard_test.py

```diff
@@ -0,0 +1,144 @@
+import asyncio
+import time
+from typing import Dict, Optional
+from pprint import pprint
+
+import requests
+import ray
+import logging
+
+from collections import defaultdict
+from ray._private.test_utils import format_web_url, fetch_prometheus_metrics
+from ray.util.scheduling_strategies import NodeAffinitySchedulingStrategy
+from pydantic import BaseModel
+from ray.dashboard.consts import DASHBOARD_METRIC_PORT
+
+logger = logging.getLogger(__name__)
+
+
+class Result(BaseModel):
+    success: bool
+    # endpoints -> P95 latency
+    p95_ms: Optional[Dict[str, float]]
+    # endpoints -> P95 latency
+    p50_ms: Optional[Dict[str, float]]
+    # Dashboard memory usage in MB.
+    memory_mb: Optional[float]
+
+
+# Currently every endpoint is GET endpoints.
+endpoints = [
+    "/logical/actors",
+    "/nodes?view=summary",
```

Comment:
Does it make sense to track query time per endpoint? With this, we will have a graph per endpoint and track p95 across multiple tests. 

Other options

1. Aggregate all P95
2. Choose only a couple of endpoints that will be slow for the workload.

Reply:
Why don't we collect this data and log it, but only emit a single p95 for the perf dashboard? That way the regression tracking is simple, but we can still have more granular data in the logs if needed.

Reply:
This sounds good to me (so it is option 1).